packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -373,3 +374,58 @@ test.describe('`isDismissible` prop', () => {
         });
     });
 });
+
+test.describe('isOpen prop', () => {
```

Comment:
I think some tests checking the scroll enabling logic would be useful here.

Ideally, we want to check that scrolling is re-enabled when:
- the modal is closed
- the modal is removed from the DOM

Reply:
@jamieomaguire By this comment, when you say the modal is removed from the DOM, do you mean that it existed at the beginning of the test and we're just saying that `isOpen: false`? Or do you mean that we never loaded the modal into the dom as part of the test wrapper in the first place? 🤔